docs/api/methods.jmd

```diff
@@ -313,6 +313,20 @@ A `resizeInfo` object, which will be passed to the supplied function, contains t
 
 ]) }}
 
+{{ api_method("getParams", "getParams (id)", "Get the parameters for an upload request. Pass in a file id to make the parameters specific to that file.",
+[
+    {
```

Comment:
@seltix5 this object should also be removed from here and return statement needs to return an object
Currently you have `null` there, it should be like:

```
[
    {
        "type": "Object",
        "description": "Object containing parameters for the upload request"
    }

]
```